cfy_manager/components/manager/config/cloudify_manager.te

```diff
@@ -0,0 +1,27 @@
+module cloudify_manager 1.0;
+
+require {
+        type httpd_t;
+        type unreserved_port_t;
+        type ntop_port_t;
+        class tcp_socket name_connect;
+
+        type postgresql_port_t;
+        type haproxy_t;
+        type gatekeeper_port_t;
+        class tcp_socket { name_bind name_connect };
+}
+
+#============= httpd_t ==============
+
+# ntop_port_t for `node` on :3000
+allow httpd_t ntop_port_t:tcp_socket name_connect;
+
+# unreserved_port_t for `node` on :8088 and `gunicorn` on :8100
+allow httpd_t unreserved_port_t:tcp_socket name_connect;
+
+#============= haproxy_t ==============
```

Comment:
haproxy section can go if this is likely to take longer than a day to finish and get merged as we'll hopefully be rid of it by tomorrow.